Create ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md (#2081)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md

---
date: ""

> title: Create a type dashboard list view (#5975)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Add dashboard list view
```

> title: Create a type dashboard list view (#5975)

Reviewer: ```suggestion
title: Add Dashboard manager feature with list of type dashboards
```

> title: New dashboard manager with list of device type dashboards

Reviewer 2: @janhommes  I assume that documentation will also be added shortly?

> title: Add Dashboard manager feature with list of type dashboards

Reviewer 2: ```suggestion
title: New dashboard manager with list of device type dashboards
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58509
version: 1020.0.0
---

> Create a type dashboard list view (#5975)

Author: The Claude AI suggested to use the following body:
```suggestion
To provide users with an overview of their dashboards, a new list view has been added. The dashboard list view displays all dashboards in a tabular format, showing relevant information such as the dashboard name, description, owner, and last modification date. This change enhances the user experience by making it easier to browse, search and manage multiple dashboards in one central place.
```

> Create a type dashboard list view (#5975)

Reviewer: Add Dashboard manager view. For now it contains only list of device type dashboards. It aggregates all dashboards that has device type assigned, it allows to remove these dashboards and it is also possible to view all the devices that has access to particular dashboard from list. User can edit dashboard by navigating to device view.

> Create a type dashboard list view (#5975)

Reviewer: ```suggestion
Add Dashboard manager view. For now it contains only list of device type dashboards. It aggregates all dashboards that has device type assigned, it allows to remove these dashboards and it is also possible to view all the devices that has access to particular dashboard from list. User can edit dashboard by navigating to device view.
```

> Add Dashboard manager view. For now it contains only list of device type dashboards. It aggregates all dashboards that has device type assigned, it allows to remove these dashboards and it is also possible to view all the devices that has access to particular dashboard from list. User can edit dashboard by navigating to device view.

Reviewer 2: ```suggestion
A dashboard manager view has been added. This dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned. Dashboards can be removed via the list. Moreover you can view all devices that have access to a particular dashboard. Dashboards can be edited by navigating to the device view.
```

> A dashboard manager view has been added. This dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned. Dashboards can be removed via the list. Moreover you can view all devices that have access to a particular dashboard. Dashboards can be edited by navigating to the device view.

Reviewer 3: Proposing this description : A dashboard manager view has been added to streamline the management of device type dashboards. This view aggregates and displays a list of all dashboards assigned to specific device types. From this list, dashboard templates linked to device types can be viewed ,edited by navigating to a particular instance, and templates can also be deleted if needed.

Reviewer: @rahultsag  your description is commited now

> A dashboard manager view has been added. This dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned. Dashboards can be removed via the list. Moreover you can view all devices that have access to a particular dashboard. Dashboards can be edited by navigating to the device view.

Reviewer: ```suggestion
A dashboard manager view has been added to streamline the management of device type dashboards. This view aggregates and displays a list of all dashboards assigned to specific device types. From this list, dashboard templates linked to device types can be viewed, edited by navigating to a particular instance and templates can also be deleted if needed.
```
